src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,58 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator
+{
+    private static final String DELIMITER = ",|:";
+    private static final int DELIMITER_GROUP_ID = 1;
+    private static final int STRING_GROUP_ID = 2;
+
+    public int add(String str){
+
+        if(isBlank(str)){
+            str = "0";
+        }
```

Comment:
```suggestion
        if (isBlank(str)) {
            return 0;
        }
```
예외 케이스는 곧바로 리턴해도 좋겠네요!